cache-filter/src/filter/http.rs

```diff
@@ -301,7 +306,7 @@ impl Context for CacheFilter {
                             request_process_failure(self, self)
                         }
                     }
-                    Ok(Authorization::Denied(denied_auth)) => {
+                    Ok(Authorization::Error(denied_auth)) => {
                         info!("authorization was denied with code: {}", denied_auth.code());
                         request_process_failure(self, self);
                         return;
```

Comment:
This is not a denied authorization, but an error in the authorization call - ie. missing application identifier/user_key, missing service identifier or wrong service token, specified a missing metric, etc.

Reply:
ops. last moment change of the threescalers api. will fix that info log.

Reply:
Also, change the variable name above it.

Reply:
fixed